drivers/bluetooth/hci/Kconfig.siliconlabs

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2024 SiliconLabs <www.silabs.com/>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_SILABS_EFR32_BUFFER_MEMORY
+	int "Silicon Labs Bluetooth Library memory buffer size"
+	depends on BT_SILABS_EFR32
+	default 6144
+	help
+	  Select the size of allocated memory buffer for the Silicon Labs
+	  Bluetooth Library.
+
+config BT_SILABS_EFR32_USER_ADVERTISERS
+	int "Silicon Labs Bluetooth Library user advertisement sets"
```

Comment:
I believe you won't have to repeat `Silicon Labs Bluetooth Library` if you use `menuconfig`.